src/index.html

```diff
@@ -12,7 +15,50 @@
       href="./style.css"
     />
   </head>
-  <body>
-    <h1>Stars</h1>
+  <body class="page__body">
+    <main class="main">
+      <div class="stars stars--0">
+        <span class="stars__star"></span>
+        <span class="stars__star"></span>
+        <span class="stars__star"></span>
+        <span class="stars__star"></span>
+        <span class="stars__star"></span>
+      </div>
+      <div class="stars stars--1">
```

Comment:
Remember about correct indentation between parent and child elements. Each level of nesting, including text, contained inside the element, requires 2-space offset. Also, blank lines shouldn't be between parent and child elements.